study.md

```diff
@@ -43,7 +44,7 @@ Although both Array and Enumerable have a `count` method, Array also defines the
 method `length`.  Why is `length` sensibly defined on Array but not Enumerable?
 
 ```md
-<!-- your answer here -->
+<!-- Length gives the amount of elements in an array, where as count gives the amount of items in an array, including if an element is made up of more then one item.  -->
```

Comment:
This does not answer the question.

`length` makes sense on `Array` because arrays have a finite length.  It does not make sense for an `Enumerable` to have a `length` because it might not be finite.  For example, an enumerable that returns each digit of π on an iteration has does not have a finite length.